lib/nationbuilder/client.rb

```diff
@@ -44,6 +45,8 @@ def raw_call(path, method, body = {}, args = {})
       }
     }
 
+    request_args[:query][:fire_webhooks] = @fire_webhooks unless @fire_webhooks.nil?
```

Comment:
Sorry misunderstood. Ok, let me rewrite it using your opts argument + applying it per call.